src/id3/v2/tag.rs

```diff
@@ -349,6 +376,14 @@ fn new_picture_frame(picture: Picture, flags: FrameFlags) -> Frame<'static> {
 	}
 }
 
+fn merge_num_pair<N, T>(number: N, total: T) -> String
```

Comment:
- `format_number_pair()`? No need for abbreviations here.
- Using optional arguments would eliminate redundant code in setters and make this function more versatile **and** testable in isolation.

Reply:
I fixed this part.